### PR TITLE
Prevent Quote Message Overflow

### LINKED ISSUE #658: Quoted messages overflow on long messages
### Description:

When quoting long messages, the quoted message container overflows outside the visible area,

### Steps to reproduce:

1. Send a long message
2. Try quoting this message 
3. Observe that the quoted message content overflows outside the visible area.

### Expected behavior:

The quoted message container should wrap long text content without overflowing, ensuring the entire message is visible.

### Actual behavior:

[before_fix.webm](https://github.com/user-attachments/assets/6ac47204-3022-4dcd-8f20-a77dfb70659f)

### PR DESCRIPTION
**Pull request by abirc8010**
# Brief Title

Added max-width for the container and wrapped the text content

## Acceptance Criteria fulfillment

- [ ] Stop overflow of the message content

Fixes #658 

## Video/Screenshots

[after_fix.webm](https://github.com/user-attachments/assets/b89099a8-5293-4d03-8fb7-e95d609924c3)

**Note**: The PR will be ready for live testing at https://rocketchat.github.io/EmbeddedChat/pulls/pr-659 after approval. Contributors are requested to replace `<pr_number>` with the actual PR number.

### CODE REVIEW

**Comment by abirc8010** [comment]
Hey @Spiral-Memory could you please review this PR ?

File: packages/react/src/views/ChatInput/ChatInput.js [hunk 1]
```diff
@@ -422,7 +425,13 @@ const ChatInput = ({ scrollToBottom }) => {
   };
 
   return (
-    <Box className={`ec-chat-input ${classNames}`} style={styleOverrides}>
+    <Box
+      className={`ec-chat-input ${classNames}`}
+      style={styleOverrides}
+      css={css`
+        width: ${showSidebar ? 'calc(100vw - 350px - 3rem)' : '98vw'};
```
Comment (Spiral-Memory) [inline packages/react/src/views/ChatInput/ChatInput.js:432]: Can we do anything better than this ? Like any other approach, we can fix this ?

Comment (abirc8010) [reply packages/react/src/views/ChatInput/ChatInput.js]: @Spiral-Memory , I changed the approach.

**Review by Spiral-Memory — APPROVED** [review]

**Comment by Spiral-Memory** [comment]
Looks great to me. 
Thanks for the fix @abirc8010

**Pull request merged**